spec/features/write_to_councillors_spec.rb

```diff
@@ -160,11 +160,10 @@
       open_email("louise@council.nsw.gov.au")
 
       expect(current_email).to_not have_content("For the attention of the General Manager / Planning Manager / Planning Department")
-      expect(current_email).to have_content("Hi Louise Councillor")
```

Comment:
Do you think we should put the councillor name expectation back so we're checking their name is included in the message?